style.css

```diff
@@ -405,3 +448,229 @@ textarea::placeholder {
   place-items: center;
   margin-top: 55px;
 }
+
+@media screen and (min-width: 768px) {
+  .landing-page {
+    width: 1585px;
+    margin: auto;
+    height: 981px;
+    border-radius: 0 0 0 70px;
+  }
+
+  .header-nav {
+    width: 100%;
+    background-color: #fff;
+    display: flex;
+    flex-direction: row;
+    position: fixed;
+    justify-content: space-between;
+    align-items: center;
+    top: 0;
+  }
+
+  .logo {
+    position: relative;
+    margin-bottom: 70px;
+  }
+
+  .logo-text {
+    text-decoration: none;
+  }
+
+  .menu {
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .nav-items {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    position: relative;
+    bottom: 50px;
+    right: 120px;
+    gap: 15px;
+  }
+
+  .nav-link {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-decoration: none;
+    justify-content: space-between;
+  }
+
+  #hamburger {
+    display: none;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-image: url("./assets/images/header-bg.png");
+    background-repeat: no-repeat;
+    background-position: 20px 0;
+    height: 825px;
+    width: 1585px;
```

Comment:
- [ ] As I mentioned the reasons earlier, kindly consider removing the exact widths on your containers.